src/components/course/EnrollButton.jsx

```diff
@@ -140,7 +124,7 @@ export default function EnrollButton() {
     if (isUserEnrolled) {
       return (
         <a
-          className="btn btn-success btn-block rounded-0 py-2"
+          className={enrollLinkClass}
           href={isCourseStarted
             ? `${process.env.LMS_BASE_URL}/courses/${key}/info`
             : `${process.env.LMS_BASE_URL}/dashboard`}
```

Comment:
Not necessarily related to your actual PR, but it looks like if the user is already enrolled and the course has NOT started, the enroll button currently links to b2c dashboard.

This should likely link to the b2b dashboard page. With that, I might suggest keeping the anchor link when `isCourseStarted` is true but render a `Link` from react-router that goes to `/{enterprise_slug}` to take advantage of React's routing.

Reply:
Updated